Modules/Loadable/Segmentations/Widgets/qMRMLSegmentsModel.cxx

```diff
@@ -241,6 +241,7 @@ void qMRMLSegmentsModel::setSegmentationNode(vtkMRMLSegmentationNode* segmentati
     d->SegmentationNode->AddObserver(vtkSegmentation::SegmentModified, d->CallBack, -10.0);
     d->SegmentationNode->AddObserver(vtkSegmentation::SegmentsOrderModified, d->CallBack, -15.0);
     d->SegmentationNode->AddObserver(vtkMRMLDisplayableNode::DisplayModifiedEvent, d->CallBack, -15.0);
+    d->SegmentationNode->AddObserver(vtkMRMLSegmentationNode::SegmentationChangedEvent, d->CallBack, 10.0);
```

Comment:
Could you add a comment on why we cannot use default priority? (it would be nice to document it for the other events as well)

Reply:
Default priority is fine in this case. I'm not sure if or why it is required for the other events.